### DISCUSSION TITLE
GPU recommendations & guidelines?

### DISCUSSION THREAD

**Discussion by stevekm** [NVlabs/instant-ngp#818] [General]
It seems that the examples shown in the instant-ngp repo are all rendered using the RTX 3090 video card. Other users have encountered errors with cards with less memory or older cards (example https://github.com/NVlabs/instant-ngp/discussions/232).

Are there any formal guidelines on what the GPU requirements for use with this project under different input conditions? Are there any more economical options for NeRF rendering with older cards, cheaper cards, or maybe cloud training and rendering instances?

I was considering getting a RTX 3070 to try this out but it seems they only come with 8GB  of memory. However a 3090 (with 24GB RAM) is a much larger financial investment.

**Comment by stevekm** [comment]
Note that there are several GPU instance types available with AWS here; https://docs.aws.amazon.com/dlami/latest/devguide/gpu.html

Are any of these able to run the `instant-ngp` software? And is it possible to do all of the training on a remote instance like this and then pull down the final result for viewing in the GUI locally?

**Reply by oscarqht** [reply]
Regarding the "is it possible to do all of the training on a remote instance" part, you can take a look at [nerfstudio](https://github.com/nerfstudio-project/nerfstudio#quickstart), they provided a [Colab notebook](https://colab.research.google.com/github/nerfstudio-project/nerfstudio/blob/main/colab/demo.ipynb) that allows you to run nerf on remote server and train against some predefined dataset or your own photos/video.

**Comment by oscarqht** [comment]
I don't have a Windows PC nor a Nvidia GPU with me but really want to try instant-ngp. I'm thinking of launching an EC2 instant on AWS which they do have GPU support but not sure whether it is compatible with this repo. Thus I'm also looking for some tutorials or guidelines on whether it is possible to run instant-ngp on a cloud server and if yes, how should I set the server up.
